IPs/RSKIP_tbd.md

```diff
@@ -0,0 +1,282 @@
+
+#### New Powpeg funding
+
+Once the SVP starts, the Bridge will send the minimum required satoshis to the new Powpeg address. It should be enough for the new Powpeg to be able to send back a transaction. The size of the Proof transaction will vary depending on the size of the new Powpeg input script.
+
+The process consists of the following 3 steps.
+
+##### Funding transaction creation
+
+As soon as a new Powpeg has been committed, the Bridge will create a SPV funding transaction for the new Powpeg.
+
+This transaction is similar to a peg-out where the recipient is the new Powpeg Bitcoin address, and the amount to receive is calculated based on the minimum amount of satoshis required to spend a UTXO by the new Powpeg.
+
+** :warning: TODO: HACER CALCULOS REALES**
+
+Possible calculation:
+```
+NUMBER_OF_SIGNATORIES => NS
+(NS * 73 + 600) * FEE_PER_BYTE
+
+Given 9 signatories and a feePerKb of 30_000 satoshis:
+37710 satoshis are to be sent to the new Powpeg
+```
+
+Once the transaction is created, it has to be stored in the set for peg-outs waiting for confirmations (i.e. "releaseTransactionSet").
+At the same time a new value in the storage should be added to identify this special kind of transaction.
+
+** :warning: TODO: DEBERIA CONSIDERAR MULTIPLES REGISTROS?**
+Suggested implementation:
+```
+KEY: "SVP_FUNDING_TX"
+VALUE: sighash of SVP funding tx
+```
+
+##### Funding transaction confirmation
+
+The SVP funding transaction goes through the usual peg-out confirmation process. Once the transaction accumulates at least 4_000 Rootstock blocks (for mainnet) it should be ready to be moved from the current set to the waiting for signatures set (i.e. "rskTxsWaitingFS").
+
+##### Funding transaction signing
+
+The funding transaction should be signed as any other peg-out. Once it is fully signed it should be removed from the set and an release_btc event should be emitted.
+From now on, the funding transaction should be broadcasted to Bitcoin, and the new Powpeg will be responsible for registering in the Bridge, and create the Proof.
+
+### Proof transaction creation
+
+For the new Powpeg to be able to proof its spendability, the funding needs to be registered, and once registered the Proof will be created, confirmed, signed and broadcasted, for the current Powpeg to validate it.
+
+#### Funding transaction registration
+
+The funding registration should be registered through a new Bridge method, dedicated specifically for SVP operations.
+
+New Bridge method signature:
+```
+registerSVPBitcoinTransaction(rawBtcTx bytes, blockHeight uint256, partialMerkleTree bytes) returns uint8
+```
+
+This method will be publicly callable, but only from EOA addresses.
+It will perform a number of verifications:
+- SVP period is still enabled (i.e. CURRENT_RSK_BLOCK - NEW_POWPEG_CREATION_BLOCK < SVP_PERIOD) ** :warning: TODO: VALIDAR!!!**
```

Comment:
I thinks this should be correct. 
When we commit the change, the NEW_POWPEG_CREATION_BLOCK is set. And right after, the validation phase starts, which lasts SVP_PERIOD. 
So the CURRENT_RSK_BLOCK when we call the _registerSVPBitcoinTransaction_ should be before the (change commit plus the svp_period)
i.e., CURRENT_RSK_BLOCK < NEW_POWPEG_CREATION_BLOCK + SVP_PERIOD
that is exactly what is written